### PR TITLE
Update UlozTo.py

### PR DESCRIPTION
**Pull request by Poslik**
... this part is not needed any more.

### CODE REVIEW

File: src/pyload/plugins/downloaders/UlozTo.py [hunk 1]
```diff
@@ -147,11 +147,11 @@ def handle_free(self, pyfile):
                 xapca = self.load("https://ulozto.net/reloadXapca.php",
                                   get={'rnd': timestamp()})
 
-                xapca = xapca.replace(
-                    'sound":"',
-                    'sound":"https:').replace(
-                    'image":"',
-                    'image":"https:')
+                #xapca = xapca.replace(
```
Comment (GammaC0de) [inline src/pyload/plugins/downloaders/UlozTo.py:150]: If its not needed then delete, do not comment.

**Comment by GammaC0de** [comment]
Please increase plugin vesion

**Comment by danoslas** [comment]
Hi, I believe this change has already been made in pull request #4176.
Only it was not merged into the `main`, but into the `develop` branch.

**Comment by GammaC0de** [comment]
merged develop branch into main.

**Pull request closed without merging**